_posts/2020-01-30-Age-Targeting-Part1.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Increase your target audience via ML on Spark [part 1]
+subtitle: Ground truth selection
+category: data
+tags: [adtech, howto, spark]
+author: Alessio Borgheresi
+header-img: "images/Age-Targeting/true_age.jpg"
+---
+
+
+In mobile advertising it is possible to target the audience of the campaign based on some charateristics of the users.
```

Comment:
"to target the audience of the campaign based" -> to target specific audiences based